### PR TITLE
footer july 2022

### PR DESCRIPTION
**Pull request by ahmadabdolsaheb**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: build/universal/trending/english.yaml [hunk 1]
```diff
@@ -1,60 +1,60 @@
...
+  article25title: "Declare an Array in Java"
+  article25link: "https://www.freecodecamp.org/news/java-array-how-to-declare-and-initialize-an-array-in-java-example/"
+  article26title: "Get Current Time in Python"
+  article26link: "https://www.freecodecamp.org/news/python-get-current-time/"
+  article27title: "Change Div Background Color"
+  article27link: "https://www.freecodecamp.org/news/div-background-color-how-to-change-background-color-in-css/"
+  article28title: "Get Variable Type in Python"
+  article28link: "https://www.freecodecamp.org/news/python-print-type-of-variable-how-to-get-var-type/"
+  article29title: "Logical Operators in Python"
+  article29link: "https://www.freecodecamp.org/news/operators-in-python-how-to-use-logical-operators-in-python/"
```
Comment (ahmadabdolsaheb) [inline build/universal/trending/english.yaml:60]:
```suggestion
article0title: "Python Parse JSON"
article0link: "https://www.freecodecamp.org/news/python-parse-json-how-to-read-a-json-file/"
article1title: "HTML Cheat Sheet "
article1link: "https://www.freecodecamp.org/news/html-cheat-sheet-html-elements-list-reference/"
article2title: "ROW_NUMBER in SQL"
article2link: "https://www.freecodecamp.org/news/row_number-in-sql-select-top-example-in-sql-and-sql-server2/"
article3title: "HTML Center Image"
article3link: "https://www.freecodecamp.org/news/html-center-image-css-align-img-center-example/"
article4title: "JavaScript Replace"
article4link: "https://www.freecodecamp.org/news/javascript-replace-how-to-use-the-string-prototype-replace-method-js-example/"
article5title: "HTML HR Tag Example"
article5link: "https://www.freecodecamp.org/news/html-horizontal-line-hr-tag-example/"
article6title: "JavaScript UpperCase"
article6link: "https://www.freecodecamp.org/news/javascript-uppercase-how-to-capitalize-a-string-in-js-with-touppercase/"
article7title: "Ordered List in HTML"
article7link: "https://www.freecodecamp.org/news/ordered-list-in-html-ol-tag-example/"
article8title: "Screenshot on Windows"
article8link: "https://www.freecodecamp.org/news/how-to-screenshot-on-windows-take-a-screen-shot-on-pc/"
article9title: "Sort an Array in Java"
article9link: "https://www.freecodecamp.org/news/java-sort-array-how-to-reverse-an-array-in-ascending-or-descending-order-with-arrays-sort-2/"
article10title: "HTML Video"
article10link: "https://www.freecodecamp.org/news/html-video-how-to-embed-a-video-player-with-the-html-5-video-tag/"
article11title: "CSS Overflow"
article11link: "https://www.freecodecamp.org/news/css-overflow-visible-scroll-auto-hidden/"
article12title: "What is REST?"
article12link: "https://www.freecodecamp.org/news/what-is-rest-rest-api-definition-for-beginners/"
article13title: "VGA No Signal"
article13link: "https://www.freecodecamp.org/news/vga-no-signal-how-to-fix-a-monitor-connection-on-windows-10-pc/"
article14title: "CSS Text Align"
article14link: "https://www.freecodecamp.org/news/css-text-align-centered-justified-right-aligned-text-style-example/"
article15title: "HTML Label Tag"
article15link: "https://www.freecodecamp.org/news/html-label-label-tag-example/"
article16title: "CSS Button Style"
article16link: "https://www.freecodecamp.org/news/css-button-style-hover-color-and-background/"
article17title: "For Loop in Java"
article17link: "https://www.freecodecamp.org/news/for-loop-in-java-foreach-loop-syntax-example/"
article18title: "Linux cp Command"
article18link: "https://www.freecodecamp.org/news/the-linux-cp-command-how-to-copy-files-in-linux/"
article19title: "Link CSS to HTML"
article19link: "https://www.freecodecamp.org/news/how-to-link-css-to-html/"
article20title: "Cast a Function in SQL"
article20link: "https://www.freecodecamp.org/news/cast-a-function-in-sql-convert-char-to-int-sql-server-example/"
article21title: "Java Logical Operators"
article21link: "https://www.freecodecamp.org/news/java-operator-and-or-logical-operators/"
article22title: "Chmod Command in Linux"
article22link: "https://www.freecodecamp.org/news/how-to-change-file-permissions-with-the-chmod-command-on-linux/"
article23title: "Python Get Last Element"
article23link: "https://www.freecodecamp.org/news/python-get-last-element-in-list-how-to-select-the-last-item/"
article24title: "Python Add to Dictionary"
article24link: "https://www.freecodecamp.org/news/python-add-to-dictionary-adding-an-item-to-a-dict/"
article25title: "Declare an Array in Java"
article25link: "https://www.freecodecamp.org/news/java-array-how-to-declare-and-initialize-an-array-in-java-example/"
article26title: "Get Current Time in Python"
article26link: "https://www.freecodecamp.org/news/python-get-current-time/"
article27title: "Change Div Background Color"
article27link: "https://www.freecodecamp.org/news/div-background-color-how-to-change-background-color-in-css/"
article28title: "Get Variable Type in Python"
article28link: "https://www.freecodecamp.org/news/python-print-type-of-variable-how-to-get-var-type/"
article29title: "Logical Operators in Python"
article29link: "https://www.freecodecamp.org/news/operators-in-python-how-to-use-logical-operators-in-python/"
```

Comment (ahmadabdolsaheb) [inline build/universal/trending/english.yaml:60]:
```suggestion
article0title: "Python Parse JSON"
article0link: "https://www.freecodecamp.org/news/python-parse-json-how-to-read-a-json-file/"
article1title: "HTML Cheat Sheet "
article1link: "https://www.freecodecamp.org/news/html-cheat-sheet-html-elements-list-reference/"
article2title: "ROW_NUMBER in SQL"
article2link: "https://www.freecodecamp.org/news/row_number-in-sql-select-top-example-in-sql-and-sql-server2/"
article3title: "HTML Center Image"
article3link: "https://www.freecodecamp.org/news/html-center-image-css-align-img-center-example/"
article4title: "JavaScript Replace"
article4link: "https://www.freecodecamp.org/news/javascript-replace-how-to-use-the-string-prototype-replace-method-js-example/"
article5title: "HTML HR Tag Example"
article5link: "https://www.freecodecamp.org/news/html-horizontal-line-hr-tag-example/"
article6title: "JavaScript UpperCase"
article6link: "https://www.freecodecamp.org/news/javascript-uppercase-how-to-capitalize-a-string-in-js-with-touppercase/"
article7title: "Ordered List in HTML"
article7link: "https://www.freecodecamp.org/news/ordered-list-in-html-ol-tag-example/"
article8title: "Screenshot on Windows"
article8link: "https://www.freecodecamp.org/news/how-to-screenshot-on-windows-take-a-screen-shot-on-pc/"
article9title: "Sort an Array in Java"
article9link: "https://www.freecodecamp.org/news/java-sort-array-how-to-reverse-an-array-in-ascending-or-descending-order-with-arrays-sort-2/"
article10title: "HTML Video"
article10link: "https://www.freecodecamp.org/news/html-video-how-to-embed-a-video-player-with-the-html-5-video-tag/"
article11title: "CSS Overflow"
article11link: "https://www.freecodecamp.org/news/css-overflow-visible-scroll-auto-hidden/"
article12title: "What is REST?"
article12link: "https://www.freecodecamp.org/news/what-is-rest-rest-api-definition-for-beginners/"
article13title: "VGA No Signal"
article13link: "https://www.freecodecamp.org/news/vga-no-signal-how-to-fix-a-monitor-connection-on-windows-10-pc/"
article14title: "CSS Text Align"
article14link: "https://www.freecodecamp.org/news/css-text-align-centered-justified-right-aligned-text-style-example/"
article15title: "HTML Label Tag"
article15link: "https://www.freecodecamp.org/news/html-label-label-tag-example/"
article16title: "CSS Button Style"
article16link: "https://www.freecodecamp.org/news/css-button-style-hover-color-and-background/"
article17title: "For Loop in Java"
article17link: "https://www.freecodecamp.org/news/for-loop-in-java-foreach-loop-syntax-example/"
article18title: "Linux cp Command"
article18link: "https://www.freecodecamp.org/news/the-linux-cp-command-how-to-copy-files-in-linux/"
article19title: "Link CSS to HTML"
article19link: "https://www.freecodecamp.org/news/how-to-link-css-to-html/"
article20title: "Cast a Function in SQL"
article20link: "https://www.freecodecamp.org/news/cast-a-function-in-sql-convert-char-to-int-sql-server-example/"
article21title: "Java Logical Operators"
article21link: "https://www.freecodecamp.org/news/java-operator-and-or-logical-operators/"
article22title: "Chmod Command in Linux"
article22link: "https://www.freecodecamp.org/news/how-to-change-file-permissions-with-the-chmod-command-on-linux/"
article23title: "Python Get Last Element"
article23link: "https://www.freecodecamp.org/news/python-get-last-element-in-list-how-to-select-the-last-item/"
article24title: "Python Add to Dictionary"
article24link: "https://www.freecodecamp.org/news/python-add-to-dictionary-adding-an-item-to-a-dict/"
article25title: "Declare an Array in Java"
article25link: "https://www.freecodecamp.org/news/java-array-how-to-declare-and-initialize-an-array-in-java-example/"
article26title: "Get Current Time in Python"
article26link: "https://www.freecodecamp.org/news/python-get-current-time/"
article27title: "Change Div Background Color"
article27link: "https://www.freecodecamp.org/news/div-background-color-how-to-change-background-color-in-css/"
article28title: "Get Variable Type in Python"
article28link: "https://www.freecodecamp.org/news/python-print-type-of-variable-how-to-get-var-type/"
article29title: "Logical Operators in Python"
article29link: "https://www.freecodecamp.org/news/operators-in-python-how-to-use-logical-operators-in-python/"
```

**Comment by ahmadabdolsaheb** [comment]
@kris, @ieahleen thought you might be interested.

File: build/universal/trending/english.yaml [hunk 2]
```diff
@@ -1,60 +1,60 @@
...
-article26link: "https://www.freecodecamp.org/news/scanning-and-repairing-drive-how-to-fix-stuck-windows-10-pc-hard-drive/"
-article27title: "Color Codes for Grey Palette"
-article27link: "https://www.freecodecamp.org/news/color-shades-hex-code-and-rgb-for-gray-color-palette/"
-article28title: "Binary Search Tree Traversal"
-article28link: "https://www.freecodecamp.org/news/binary-search-tree-traversal-inorder-preorder-post-order-for-bst/"
-article29title: "RTC Connecting Discord Fix"
-article29link: "https://www.freecodecamp.org/news/rtc-connecting-discord-how-to-fix-the-server-error/"
+article0title: "Python Parse JSON"
+article0link: "https://www.freecodecamp.org/news/python-parse-json-how-to-read-a-json-file/"
+article1title: "HTML Cheat Sheet "
```
Comment (scissorsneedfoodtoo) [inline build/universal/trending/english.yaml:3]:
```suggestion
article1title: "HTML Cheat Sheet"
```

**Review by scissorsneedfoodtoo — CHANGES_REQUESTED** [review]
Just found one title with and extra whitespace at the end. Otherwise, everything LGTM 👍 👍

**Review by scissorsneedfoodtoo — APPROVED** [review]

**Pull request merged**